### PR TITLE
doc: Update CONTRIBUTING.adoc

### PR DESCRIPTION
**Pull request by NathalieC**
* add Troubleshooting section formatting guidelines

### CODE REVIEW

File: docs/content/CONTRIBUTING.adoc [hunk 1]
```diff
@@ -233,3 +231,16 @@ See https://docs.antora.org/antora/2.3/examples-directory/
 This allows user to download the source as attachments by providing a link directly for the code (no duplication between the actual examples and the documentation)
 
 Attachments are also possible: https://docs.antora.org/antora/2.3/attachments-directory/
+
+=== Troubleshooting sections
+https://documentation.bonitasoft.com/bonita/2021.2/bonita-overview/ui-designer-overview#_troubleshooting[Here is a good example] of what *Troubleshooting sections* should look like:
+* What is always here: 
+ ** The icon with a screen and a monkey wrench. xref:image:images/troubleshooting.png[Here is the link] to reference it
```
Comment (tbouffard) [inline docs/content/CONTRIBUTING.adoc:238]: broken link: please provide a link to an image in the bonita-doc repository 😸

**Review by tbouffard — CHANGES_REQUESTED** [review]

File: docs/content/CONTRIBUTING.adoc (hunk 1 above)
Comment (NathalieC) [reply docs/content/CONTRIBUTING.adoc:239]: eh oui... ok.

**Comment by NathalieC** [comment]
I've replaced the image link with an absolute link. Better?

**Review by tbouffard — APPROVED** [review]
LGTM

**Pull request merged**